README.md

```diff
@@ -33,6 +33,9 @@ If you are using Ubuntu Core 16, connect the `docker:home` plug as it's not auto
 sudo snap connect docker:home
 ```
 
+The `docker-compose` [alias](https://snapcraft.io/docs/commands-and-aliases) was set automatically for Compose V1 and remains for backwards-compatiblity.
+The recommended command-line syntax since Compose V2 is `docker compose` as described on the [Docker's official documentation](https://docs.docker.com/compose/releases/migrate/).
```

Comment:
Same link title given to two different URLs. On line 8, "Docker's official documentation" links to https://docs.docker.com, but here it links to "Migrate to Compose V2". A reader will probably think these are the same references. In general, longer sentences and more text make it more difficult to find the right information.

```suggestion
The recommended command-line syntax since Compose V2 is `docker compose` as described [here](https://docs.docker.com/compose/releases/migrate/).
```